src/whist/card.py

```diff
@@ -0,0 +1,191 @@
+"""Card related classes"""
+
+from dataclasses import dataclass
+from enum import Enum, unique
+from typing import final, NoReturn, Any
+
+
+class _OrderedEnum(Enum):
+    def __new__(cls, *args) -> NoReturn:
+        if len(args) == 1:
+            value = args[0]
+        else:
+            value = args
+
+        obj = object.__new__(cls)
+        obj._value_ = value
+        obj.__ordinal = len(cls.__members__)  # pylint: disable=protected-access
```

Comment:
```suggestion
        # pylint: disable=protected-access
        obj.__ordinal = len(cls.__members__) 
```

Reply:
why?

Reply:
I was under the impression you have to but the comment above the line for which you want to disable it.